nix/packages/moq.nix

```diff
@@ -0,0 +1,32 @@
+{ self, nixpkgs, flake-utils, ... }:
+flake-utils.lib.eachDefaultSystem (system:
+  let
+    pkgs = import nixpkgs {
+      inherit system;
+    };
+  in
+    with pkgs;
+    {
+      packages = rec {
+        moq-relay = rustPlatform.buildRustPackage rec {
+          pname = "moq-relay";
+          version = "0.6.7";
```

Comment:
Can you make this pull from Cargo.toml automatically?

Reply:
Unfortunately I can't right now. May be able to move to `crate2nix` and it can figure this stuff out at eval time

Reply:
Use `builtins.fromTOML` to extract the version field from `Cargo.toml`. No need for crate2nix to accomplish this.

Reply:
I can't manually bump this version especially because we're using release-plz. What about this snippet I found from a quick search?

```
{ pkgs ? import <nixpkgs> { } }:
let manifest = (pkgs.lib.importTOML ./Cargo.toml).package;
in
pkgs.rustPlatform.buildRustPackage rec {
  pname = manifest.name;
  version = manifest.version;
  cargoLock.lockFile = ./Cargo.lock;
  src = pkgs.lib.cleanSource ./.;
}
```

Reply:
Yup that should work

Reply:
> Use `builtins.fromTOML` to extract the version field from `Cargo.toml`. No need for crate2nix to accomplish this.

ha yeah I didn't see your reply until just now

Reply:
> > Use `builtins.fromTOML` to extract the version field from `Cargo.toml`. No need for crate2nix to accomplish this.
> 
> ha yeah I didn't see your reply until just now

`lib.importTOML` (from your snippet) is preferred here anyway because it combines reading the file + parsing, while `builtins.fromTOML` takes a string as input.

Reply:
oh TIL about `builtins.fromTOML` will update it